src/exe/cimbar_send/send.cpp

```diff
@@ -29,8 +42,6 @@ int main(int argc, char** argv)
 	    ("c,colorbits", "Color bits. [0-3]", cxxopts::value<int>()->default_value(turbo::str::str(colorBits)))
 	    ("e,ecc", "ECC level", cxxopts::value<unsigned>()->default_value(turbo::str::str(ecc)))
 	    ("f,fps", "Target FPS", cxxopts::value<unsigned>()->default_value(turbo::str::str(defaultFps)))
-	    ("r,rotatecam", "Successive images are rotated 90 degrees", cxxopts::value<bool>())
-	    ("s,shakycam", "Successive images are offset, like a shaky camera effect", cxxopts::value<bool>())
```

Comment:
Losing the shake/rotate flags -- I'm ok with that. The "shakycam" behavior is now standard.